src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Config/QueuesOptions.cs

```diff
@@ -148,5 +149,17 @@ public TimeSpan VisibilityTimeout
                 _visibilityTimeout = value;
             }
         }
+
+        public object GetLoggableOptions()
```

Comment:
I wonder if the contract shouldn't just be a Dictionary<string, string> or Dictionary<string, object> so we can avoid the json serialization?